database/mySQL/queries.js

```diff
@@ -42,7 +42,7 @@ const updateBooking = (bookingId, updatedBooking, callback) => {
   keyVals = keyVals.substring(0, keyVals.length - 1);
   
   var queryStr = `
-  UPDATE BOOKINGS 
+  UPDATE bookings 
    SET ${keyVals}
    WHERE id = ${bookingId}
    `;
```

Comment:
Your file looks clean and understandable. Only suggestion I have here is perhaps use variable names for the intended action. Example:
getListing: selectQry | postBooking: insertQry | updateBooking: updateQry.